<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserList</class>
 <widget class="QWidget" name="UserList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>聊天Chat</string>
  </property>
  <widget class="QLabel" name="picture_labal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>72</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>这是你的头像</string>
   </property>
  </widget>
  <widget class="QLabel" name="nickname_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>你的昵称</string>
   </property>
  </widget>
  <widget class="QLabel" name="sign_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>你的签名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linkMan_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索:联系人</string>
   </property>
  </widget>
  <widget class="QListWidget" name="linkmanList_listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>311</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
